extensions-support/bouncycastle/runtime/pom.xml

```diff
@@ -55,10 +55,6 @@
             <groupId>org.bouncycastle</groupId>
             <artifactId>bcpkix-jdk18on</artifactId>
         </dependency>
-        <dependency>
-            <groupId>org.javassist</groupId>
-            <artifactId>javassist</artifactId>
-        </dependency>
```

Comment:
It might be that this was the only javaassist reference in the whole source tree because of which we were modifying quarkus enforcer bans around here: https://github.com/apache/camel-quarkus/blob/main/tooling/enforcer-rules/quarkus-banned-dependencies.xsl#LL33C64-L33C102
Could you please remove `text() = 'org.javassist:javassist' or ` from there and see whether enforcer will complain about javassist somewhere else?

Reply:
I ran `mvn dependency:tree` and there are no other references to javassist. Also running the enforcer with that constraint removed didn't produce any errors.

Reply:
Perfect, thanks @djcoleman!